package.json

```diff
@@ -1,7 +1,9 @@
 {
   "author": "Francisco Tavares de Lima Neto",
   "description": "Callbag source that generates a range of numbers",
-  "dependencies": {},
+  "dependencies": {
+    "callbag-from-iter": "^1.2.0"
```

Comment:
What's the reasoning behind this? It's rather not a common practice and might cause duplicated dependencies in people's applications.

Reply:
Ok, I agree with you. On one hand I don't want to have dependencies breaking the behaviour of our implementation, on the other hand it's not good to duplicate dependencies. You can leave it as is.

Reply:
minor versions shouldnt break the backward compatibility, so it should be "safe" to use ^ instead of ~, but ofc they might because semver versions are bumped by humans (although the same argument applies to ~ - somebody might release a breaking change in patch version)